File: konjunkcio.js
// Konjunkció (ÉS) igazságtáblázata saját függvénnyel

function kon(x, y) {
  var a = Boolean(x), b = Boolean(y);
  var c = a && b;
  return Number(c);
}

console.log("A B A∧B");
console.log("-------");

for (var i = 0; i <= 1; i++) {
  for (var j = 0; j <= 1; j++) {
    var k = kon(i, j);
    console.log(i + " " + j + "  " + k);
  }
}

console.log("-------");

File: konjunkcio.py
# Konjunkció (ÉS) igazságtáblázata saját függvénnyel

def kon(x, y):
    a = bool(x)
    b = bool(y)
    c = a and b
    return int(c)

print("A B A∧B")
print("-------")

for i in range(2):       # 0 és 1
    for j in range(2):   # 0 és 1
        k = kon(i, j)
        print(f"{i} {j}  {k}")

print("-------")
